src/cfnlint/maintenance.py

```diff
@@ -58,6 +59,26 @@ def update_resource_spec(region, url):
     with open(filename, 'w') as f:
         json.dump(spec, f, indent=2, sort_keys=True, separators=(',', ': '))
 
+    lines = []
+    with open(filename, 'r') as f:
+        for line in f:
+            if 'botocore' in line:
+                service_and_type = line.split('"')[3]
+                service = '/'.join(service_and_type.split('/')[:-1])
+                botocore_type = service_and_type.split('/')[-1]
+                r = requests.get('https://raw.githubusercontent.com/boto/botocore/master/botocore/data/' + service + '/service-2.json').json()
```

Comment:
We probably don't want to pull in the `requests` dependency just for this line. Open to alternative suggestions